### PR TITLE
Updated README with group member name

### PR DESCRIPTION
**Pull request by jmrogers0602**
I was able to figure out the 2-factor authentication and have put in the pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
-Group Members:
-PI: Madicken Munk 
+
+
+Group Members: PI: Madicken Munk 
 
 Undergraduate Researchers:
- 
+
+Joseph Rogers
```
Comment (SteiSamu) [inline README.md:9]: looks good, you sure you don't want to be called Joe??

**Review by SteiSamu — CHANGES_REQUESTED** [review]
I see that there will be a conflict, could you resolve this and push again?

File: README.md (hunk 1 above)
Comment (munkm) [reply README.md:9]: Maybe you want to put your preferred name in brackets if you have an alternative? If you prefer Joseph, then no changes are necessary!

Comment (jmrogers0602) [reply README.md:9]: > looks good, you sure you don't want to be called Joe??

[Joey]

**Comment by jmrogers0602** [comment]
> I see that there will be a conflict, could you resolve this and push again?

I believe I have fixed the conflict, can you take a look?

**Review by SteiSamu — APPROVED** [review]
Looks Good! Thanks Joseph.

**Pull request merged**